src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -221,8 +232,18 @@ public void testFindAll_limitOnly() {
 
   @Test
   public void testFindAll_offsetOnly() {
-    final List<DatasetRow> newDatasetRows =
-        newDatasetRowsWith(namespaceRow.getUuid(), datasourceRow.getUuid(), 4);
+    List<DatasetRow> newDatasetRows = new ArrayList<>();
+
+    Stream.<String>builder()
+        .add("a_test_dataset")
```

Comment:
Instead of hard-coding these, it's recommended to use `CommonModelGenerator.newDatasetName()`.

Reply:
In the generation code we have, to generate a list of n of something, the pattern is `new<Object>s`. In this situation, I don't see a `newDatasetNames(limit n)`, so if you have a minute, I'd recommend adding it.